README.md

```diff
@@ -107,7 +107,7 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://math-office.onrender.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
I am highlighting these here because I can't make inline comments on the necessary lines in the _**README.md**_ file.

- [ ] Under the `Tech Stack` section, kindly remove the `Server` and `Database` subsections. No technology in this category was used for this project. 👍 
- [ ] For the `Key Features` section, kindly include 1 - 3 features that you implemented in this application. When anyone reads your _**README.md**_ file, they will easily know what this project is about 💻 
- [ ] Kindly remove the Microverse logo. If you have your own logo, you can replace the microverse logo with yours. If you don't, kindly remove it 💻